### DISCUSSION TITLE
Time +8

### DISCUSSION THREAD

**Discussion by chengidesk** [netalertx/NetAlertX#695] [Q&A]
Hi, wondering if any idea how to remove +8? 
![image](https://github.com/jokob-sk/NetAlertX/assets/155093502/0be0a741-aaed-4548-9445-2efb739113e3)

thank you !

**Comment by jokob-sk** [comment]
Hi @chengidesk ,

It's only the timezone offset. It shouldn't lead to any issues. Does it bother you?

**Comment by chengidesk** [comment]
Hi Sir,

It does because
1. It is taking a bigger screen space for dual column.
2. When I export the data, I’ll need to massage it to make good the user experience.

It is however not a show stopper.

Thanks and regards,



Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by jokob-sk** [reply]
OKay, removed the +... from the shown columns, but the export will still contain them, it's just a frontend modification: https://github.com/jokob-sk/NetAlertX/commit/302a687d41e439291c4b6df55ba52b3e5b541a16

**Reply by chengidesk** [reply]
sorry, as my daughter was down with influenza a, i was a bit busy coordinating her arrangement.

Thanks for the update, I have no idea how to make the changes to my environment :)

Wondering if I may suggest to have color for status? 
example: if the client is online, we should see status in green ("New" in color Green) but if the client is no longer online it still can be "New" in color red/amber/yellow/no color or when there is no status it will follow existing whereby it show "Online" and "blank" (offline).

At the same time i would like to suggest to have different kind of status customization for example: investigate, block (currently status value are either Online or New). I understand i could have explore field "Group" but i would actually hide "group" in future to save more "screen space".

Thank you for your support & consideration @jokob-sk !

**Reply by jokob-sk** [reply]
Thanks for the suggestion, I will think about coloring the (new) status according to the status (green - online / red - offline) . 

I'm unsure what you mean by investigate/block. If you can submit screenshots or more details as I can't understand this request.

**Reply by jokob-sk** [reply]
Also FYI you can remove columns you don't want to see to save screen space: 
![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/eff6ce3b-ad3f-4c1f-9640-74ca1190417d)

**Reply by chengidesk** [reply]
> Thanks for the suggestion, I will think about coloring the (new) status according to the status (green - online / red - offline) .
> 
> I'm unsure what you mean by investigate/block. If you can submit screenshots or more details as I can't understand this request.

what i am trying to say is to allow that to be something like "group" where it allows for custom text. however this is possibly an issue for others because they might want to export the data with this online or offline category.

**Reply by chengidesk** [reply]
> Also FYI you can remove columns you don't want to see to save screen space: ![image](https://private-user-images.githubusercontent.com/96159884/339356454-eff6ce3b-ad3f-4c1f-9640-74ca1190417d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyOTY5OTQsIm5iZiI6MTcxODI5NjY5NCwicGF0aCI6Ii85NjE1OTg4NC8zMzkzNTY0NTQtZWZmNmNlM2ItYWQzZi00YzFmLTk2NDAtNzRjYTExOTA0MTdkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDE2MzgxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0YjFmOWQ3ZWEyMzQ2MzQ0MGQ1OGFjYmJmYjhlNzhhMzA4YzRkNDY5NDdmNTI1YmI3YmNlNDg0ZDBhMzc5ZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nBcEEgLBNbvgwJSeT1kimSL8hTQ7XnKW6SXSwyGFTl4)

i actually wanted to remove the presence bar totally because once i have large numbers it doesn't really meaningful. alternatively, you can consider y axis as % . I'm looking at 500 devices in next 3 months and expecting during school days to have 150 online/presence daily.

**Reply by jokob-sk** [reply]
> > Also FYI you can remove columns you don't want to see to save screen space: ![image](https://private-user-images.githubusercontent.com/96159884/339356454-eff6ce3b-ad3f-4c1f-9640-74ca1190417d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyOTY5OTQsIm5iZiI6MTcxODI5NjY5NCwicGF0aCI6Ii85NjE1OTg4NC8zMzkzNTY0NTQtZWZmNmNlM2ItYWQzZi00YzFmLTk2NDAtNzRjYTExOTA0MTdkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDE2MzgxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk0YjFmOWQ3ZWEyMzQ2MzQ0MGQ1OGFjYmJmYjhlNzhhMzA4YzRkNDY5NDdmNTI1YmI3YmNlNDg0ZDBhMzc5ZWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nBcEEgLBNbvgwJSeT1kimSL8hTQ7XnKW6SXSwyGFTl4)
> 
> i actually wanted to remove the presence bar totally because once i have large numbers it doesn't really meaningful. alternatively, you can consider y axis as % . I'm looking at 500 devices in next 3 months and expecting during school days to have 150 online/presence daily.

You can do that already with this setting 🙂
![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/1eefddd1-5bf0-448f-ab59-5380e6455b22)

**Reply by jokob-sk** [reply]
> > Thanks for the suggestion, I will think about coloring the (new) status according to the status (green - online / red - offline) .
> > I'm unsure what you mean by investigate/block. If you can submit screenshots or more details as I can't understand this request.
> 
> what i am trying to say is to allow that to be something like "group" where it allows for custom text. however this is possibly an issue for others because they might want to export the data with this online or offline category.

Thank you for the details! I am sorry though, I'm still not following. Maybe I need a coffee 😁 When you are exporting the data via the Export CSV function, all columns, even hidden ones, are exported, so it doesn't matter where the data is stored. If you could mock up a screenshot and explain a use case that would help. 

Also in the Groups column, you can store any text you want.

**Comment by FlyingToto** [comment]
If I may jump in... 

I would see why trying to remove the time offset might be challenging to little ROI. In my limited experience, the only way to handle time stamps  is to keep everything internally in UTC. And only convert time at input/output/display UI levels. 

my 2 cents.

**Reply by jokob-sk** [reply]
Well explained 👍
